htdocs/wp-content/themes/owc-formulieren/assets/scss/elements/_arrowed-links.scss

```diff
@@ -0,0 +1,27 @@
+// List styles, WordPress default behaviour on core/list.
+.is-style-arrows {
+	li {
+		&:before {
+			color: var(--primary);
+		}
+
+		a {
+			text-decoration: none;
+		}
+	}
+}
+
+// Class added by custom code in this project for paragraphs.
+.openpdd-arrowed-links {
```

Comment:
Graag een `/elements/_paragraph.scss` toevoegen, met `p.openpdd-arrowed-links`.

Vervolgens de gedeelde SCSS properties in een @mixin zetten in `_mixins.scss`, en dit toevoegen op de `::before`'s van de lijst en paragraaf. 

```
&::before {
  @include has-arrow;
}
``